examples/telegram/no_pipeline/pictures.py

```diff
@@ -0,0 +1,154 @@
+            },
+        },
+    },
+    "pics": {
+        "ask_picture": {
+            RESPONSE: "Send me a picture",
+            TRANSITIONS: {
+                ("pics", "thank", 1.1): cnd.any(
+                    [
+                        bot.cnd.message_handler(content_types=["photo"]),
+                        bot.cnd.message_handler(func=doc_is_photo, content_types=["document"]),
+                    ]
+                ),
+                ("pics", "repeat", 0.9): cnd.true(),
+            },
+        },
+        "thank": {
+            RESPONSE: dict(
+                text="Nice! Here is my picture:",
+                # An HTTP path or a path to a local file can be used here
+                picture=kitten_url,
+            ),
+            TRANSITIONS: {("root", "fallback"): cnd.true()},
+        },
+        "repeat": {
+            RESPONSE: "I cannot find the picture. Please, try again.",
+            TRANSITIONS: {
+                ("pics", "thank", 1.1): cnd.any(
+                    [
+                        bot.cnd.message_handler(content_types=["photo"]),
+                        bot.cnd.message_handler(func=doc_is_photo, content_types=["document"]),
+                    ]
+                ),
+                ("pics", "repeat", 0.9): cnd.true(),
+            },
+        },
+    },
+}
+
+
+# %%
+actor = Actor(script, start_label=("root", "start"), fallback_label=("root", "fallback"))
+
+
+# %% [markdown]
+"""
+Although in most cases it makes sense to use only one handler to iterate over the script,
+you can always create a separate function that will perform additional tasks.
+"""
+
+
+# %%
+def extract_data(message):
+    """A function for extracting data."""
+
+    if not message.photo and not message.document:
+        return
+    photo = message.document or message.photo[-1]
+    file = bot.get_file(photo.file_id)
+    result = bot.download_file(file.file_path)
+    with open("photo.jpg", "wb+") as new_file:
+        new_file.write(result)
+
+
+# %%
+@bot.message_handler(func=lambda msg: True, content_types=content_type_media)
+def handler(update):
+    user_id = (vars(update).get("from_user")).id
+    context: Context = db.get(user_id, Context(id=user_id))
+    context = set_framework_state(context, TELEGRAM_STATE_KEY, update, inner_key="data")
+    context.add_request(vars(update).get("text", "data"))
+
+    # Extract data if present
+    if isinstance(update, types.Message):
+        extract_data(update)
+
+    context = actor(context)
+
+    db[user_id] = context
+
+    response = context.last_response
+    if isinstance(response, str):
+        bot.send_message(update.from_user.id, response)
+    elif isinstance(response, dict):
+        bot.send_message(update.from_user.id, response.get("text"))
+        with open(response.get("picture"), "rb") as file:
+            bot.send_photo(update.from_user.id, file)
+
+
+if __name__ == "__main__":
```

Comment:
We should add `and is_interactive_mode()` to condition to prevent bot from hanging automatic execution.

Reply:
example will be removed